### DISCUSSION TITLE
[Help Pls] Graduated March 2022 with Expired SDP

### DISCUSSION THREAD

**Discussion by pfei-sa** [community/community#17745] [GitHub Education]
Can someone help merge my PR @ https://github.com/education/GitHubGraduation-2022/pull/3745 ? Just graduated and cannot (and don't need) extend SDP.

**Comment by elisemoe** [comment]
Congrats on your graduation @pfei-sa ✨  We've merged your PR.

**Comment by proarafat** [comment]
I also do that [PR](https://github.com/education/GitHubGraduation-2022/pull/4820)
But what's the next step I do?
they also told me that they would send a postcard to my home but when they sed the postcard?
What's do I do now?

**Comment by totegamma** [comment]
Same situation here. My SDP just ended and no way to pass automation. https://github.com/education/GitHubGraduation-2022/pull/6389
Could you review my PR?

**Comment by Tiger-0512** [comment]
I'm in the same situation.
My graduation month is March so my SDP license has expired.
Cloud you help me? @elisemoe 

My PR is https://github.com/education/GitHubGraduation-2022/pull/7279.
